### DISCUSSION TITLE
File references are not yet supported in Studio

### DISCUSSION THREAD

**Discussion by jandresboyaca** [asyncapi/studio#576] [Q&A]
I was wondering and I was looking but I couldn't find it, why in the studio we don't allow file references?

I'll give you a bit of context:

I usually work with a lot of reference objects and recently I updated the async studio and saw this warning.
I use the reference object because I have different environments and use reference objects instead of direct references.

<img width="358" alt="image" src="https://user-images.githubusercontent.com/31944043/224830883-f1f7c936-cdb3-4260-a3d9-8b2300b1ebc3.png">

I can work but now I have to put the direct reference by URL which I can change later when I build the specs for each environment but the process takes a bit longer.

I just want to understand why this change

**Comment by iamrakesh-kumar** [comment]
same issue i am facing.

**Comment by fmvilas** [comment]
Commented in the issue #1127. Shall we discuss here or there? I'm more inclined to issues because it's an actual issue and not much of a discussion.

**Comment by ivangsa** [comment]
relative `$ref` support is coming to Studio, including full local folder editing (from the web).  see #1224
